server/opts.go

```diff
@@ -705,15 +705,17 @@ func (o *Options) processConfigFileLine(k string, v interface{}, errors *[]error
 			}
 			o.TrustedOperators = append(o.TrustedOperators, opc)
 		}
-		// In case "resolver" is defined as well, it takes precedence
-		if o.AccountResolver == nil && len(o.TrustedOperators) == 1 {
-			if accUrl, err := parseURL(o.TrustedOperators[0].AccountServerURL, "account resolver"); err == nil {
-				// accommodate nsc which appends "/accounts" during nsc push
-				suffix := ""
-				if accUrl.Path == "/jwt/v1/" || accUrl.Path == "/jwt/v1" {
-					suffix = "/accounts"
+		if len(o.TrustedOperators) == 1 {
+			// In case "resolver" is defined as well, it takes precedence
+			if o.AccountResolver == nil {
+				if accUrl, err := parseURL(o.TrustedOperators[0].AccountServerURL, "account resolver"); err == nil {
+					// nsc automatically appends "/accounts" during nsc push
+					o.AccountResolver, _ = NewURLAccResolver(accUrl.String() + "/accounts")
 				}
-				o.AccountResolver, _ = NewURLAccResolver(accUrl.String() + suffix)
+			}
+			// In case "system_account" is defined as well, it takes precedence
+			if o.SystemAccount == "" {
```

Comment:
Should we allow this to be overridden if its set in the operator JWT?

What happens when we have multiple operators and they have different ones?

If it is set, and the server config is set, should we enforce they are the same? Not just by name but the complete claim?

Reply:
How about a warning?
I left the overwrite option as an escape hatch for the multi operator case

Reply:
Possibly. For single operator if server configures a system account feels like misconfig and an error. For multiple operators the server needs to pick a system account from one of them.

Reply:
Will raise an error when that's not the case.

Reply:
I don't know the answer (did not check) but would it make sense to have all this code in "validate" code that would work for config files and when embedding. I am always conflicted because it is nice to have the error line, etc.. when getting it from config file, but we should also take into account that users embed and may configure directly through options.

Reply:
@kozlovic  That is a good point. I can move the validation code into validateTrustedOperators. We do quite a few sanity checks there already.